exporter/clickhouseexporter/README.md

```diff
@@ -0,0 +1,90 @@
+# ClickHouse Exporter
+
+**Status: experimental**
+Supported pipeline types: logs.
+
+This exporter supports sending OpenTelemetry logs to [ClickHouse](https://clickhouse.com/).
+
+Note:
+Always add [batchprocessor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor) to collector pipeline, as Clickhouse [recommend](https://clickhouse.com/docs/en/introduction/performance/#performance-when-inserting-data ) inserting data in packets of at least 1000 rows, or no more than a single request per second.
+
+## Configuration options
+
+The following settings are required:
+
+- `address` (no default): The ClickHouse server address.
```

Comment:
HTTP interface is very common and a lot of people will assume that `address` means HTTP base URL. I suggest adding a note here about address format (and/or probably give a link to the clickhouse-go repo).

Reply:
Thanks. You are right. let me add more comment for it.

Reply:
Updated. I use raw dsn for more flexibility.